uk.co.compucorp.civicrm.hrleaveandabsences/hrleaveandabsences.civix.php

```diff
@@ -342,7 +441,104 @@ function _hrleaveandabsences_civix_civicrm_alterSettingsFolders(&$metaDataFolder
   $configured = TRUE;
 
   $settingsDir = __DIR__ . DIRECTORY_SEPARATOR . 'settings';
-  if(is_dir($settingsDir) && !in_array($settingsDir, $metaDataFolders)) {
+  if (is_dir($settingsDir) && !in_array($settingsDir, $metaDataFolders)) {
     $metaDataFolders[] = $settingsDir;
   }
 }
+
+/**
+ * (Delegated) Implements hook_civicrm_entityTypes().
+ *
+ * Find any *.entityType.php files, merge their content, and return.
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_entityTypes
+ */
+
+function _hrleaveandabsences_civix_civicrm_entityTypes(&$entityTypes) {
```

Comment:
Just curious, but did this change affect anything else? Seems like changing it to return an array of entities would have some impact somewhere